custom_components/tuya_local/devices/lucci_fan.yaml

```diff
@@ -0,0 +1,23 @@
+name: Lucci Fan
+products:
+  - id: U8wU126HQlIeQHvS
+    name: Lucci Fan
```

Comment:
Would be better if you had a model number or model name in place of "Fan" here. There appear to be a couple of options for Lucci WiFi fan controllers, I don't know if they are compatible.

Reply:
This is for a retro fittable wifi module that works for multiple fans. The specific model can be found here: https://www.beaconlighting.com.au/lucci-connect-wi-fi-fan-remote-control?srsltid=AfmBOoqKSruEWAKwDguopQFpkmYI7z3aJoI2qX6L0oWp8k7zTXmvyYML